app/ancillaryFactory/src/components/NoAncillaryMessage.js

```diff
@@ -0,0 +1,15 @@
+// @flow strict
+
+import * as React from 'react';
+import { View } from 'react-native';
+import { Text, Translation } from '@kiwicom/mobile-shared';
+
+const NoAncillary = () => (
+  <View>
+    <Text>
```

Comment:
You don't need to wrap `<Translation />` in text unless you want to add some styles.